README_nextflow.md

```diff
@@ -17,12 +17,12 @@ conda activate variant-remapping
 pip install -r requirements.txt
```

Comment:
That's a good suggestion, I'll do that.

Reply:
I've reorganised the file in the repository and changed the documentation. 
I've left the nextflow pipeline in the top level directory and renamed it `main.nf` because that's a pattern I've seen use in many nextflow pipeline.

Reply:
Now it looks great, thanks!